addons/addon-base/packages/services/lib/db-password/db-password-service.js

```diff
@@ -45,10 +46,27 @@ class DbPasswordService extends Service {
     }
   }
 
+  async saveRootPassword(requestContext, { password, uid }) {
+    const username = this.settings.get('rootUserName');
+    if (!isSystem(requestContext)) {
+      throw this.boom.badRequest("'root' password can not be changed", true);
```

Comment:
cannot be changed by non-system user?

Reply:
For now it's only being used by the post deployment lambda, which is a system user. Therefore let's restrict it to only `systems` and reevaluate if we need to expand the scope in the future.

Reply:
no, I mean the comment :)